products/proxmox-ve.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Proxmox VE
+category: os
+iconSlug: proxmox
+permalink: /proxmox-ve
+alternate_urls:
+-   /pve
+-   /proxmoxve
+activeSupportColumn: Supported
```

Comment:
Not needed I think: Proxmox VE team does not seem to distinguish "active support" (bug fixes, new features...) and security support. So only the mandatory "EOL" column (named "Support" by default) is needed.